styles.css

```diff
@@ -30,17 +31,30 @@ P {
   display: inline-block;
 }
 .TweetButton {
+  margin-top: 25px;
+  margin-bottom: 50px;
   width: 200px;
+  height: 50px;
   border-top-left-radius: 50px;
   border-bottom-left-radius: 50px;
   border-top-right-radius: 50px;
   border-bottom-right-radius: 50px;
   background-color: #1c92da;
+  border: none;
+  cursor: pointer;
 }
 .TweetButton,
 Text {
-  padding: 20px;
+  padding-left: 30px;
+  padding-right: 30px;
+  padding-top: 10px;
+  padding-bottom: 10px;
```

Comment:
FYI this is equivalent to:

```css
padding: 10px 30px;
```

See https://developer.mozilla.org/en-US/docs/Web/CSS/padding